boolean-type/lorem-ipsum-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "KnpU\\LoremIpsumBundle\\KnpULoremIpsumBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "KNPU_LOREM_IPSUM_FOO": "hello",
+        "KNPU_LOREM_IPSUM_BAR": "world"
```

Comment:
What are these environment variables for?

Reply:
It was only for test: I just wanted to see, if variables are set after bundle installation.
I will handle it soon if needed.

Reply:
Ok, I've decided not to change bundle code to use those variables. There is possible for user to use them instead. For example, in subscriber, which allows to change data returned by API:

<pre>
public static function getSubscribedEvents()
{
    return [
        KnpULoremIpsumEvents::FILTER_API => 'onFilterApi',
    ];
}

public function onFilterApi(FilterApiResponseEvent $event)
{
    $data = $event->getData();
    $data['message'] = 'Have a magical day!';
    $data['hello_world'] = $_ENV['KNPU_LOREM_IPSUM_FOO'] . ', ' . $_ENV['KNPU_LOREM_IPSUM_BAR'];
    $event->setData($data);
}
</pre>